docs/_temp/Installation.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "How to Organize Devlake Projects"
+sidebar_position: 1
+description: >
+  How to Organize Devlake Projects
+---
+
+## 1. Introduction
+A typical team of developers works with `pull requests`, `deployments`, and `issues` (a.k.a. Bugs for deployment).
+
+Based on such, we want to measure their productivity and stability. This is how [DORA](docs/DORA.md) does that:
+- Productivity:
+  - How many times does the team `deploy`? (a.k.a. [Deployment Frequency](docs/Metrics/DeploymentFrequency.md))
+  - How fast are the `pull requests` resolved? (a.k.a. [Lead Time](docs/Metrics/LeadTimeForChanges.md))
+- Stability:
+  - How many `issues` per `deploys` does the team have? (a.k.a. [Change Failure Rate](docs/Metrics/CFR.md))
+  - How fast are these `issues` solved? (a.k.a. [Median Time to Restore](docs/Metrics/MTTR.md))
+
+All these questions/metrics are based on either `pull requests`, `deployments`, or `issues`.
+
+But when we scale this up, few problems arise:
+- A team usually works with multiple repositories
+- A team also might work on different projects, and we want to measure these projects separately (e.g. it is not the same to work on a big old legacy than on a greenfield)
+- There may be multiple teams
+
+This is where the `project` concept comes to play.
+
+## 2. What is a DevLake project?
+In the real world, a project is something being built and/or researched to solve some problem or to open new grounds.
+In software development, a project is just a grouping of something. In DevLake, a `project` is a grouping of `pull requests`, `deployments`, or `issues`.
+
+![](project_simple.png)
+
+## 3. As a team lead, how many DevLake projects do I need?
+
+Because of its simplicity, the concept is flexible: you decide how to arrange `pull requests`, `deployments`, and `issues`
+either by your specific projects, by teams, technology, or any other way.
+
+Let's consider an example:
```

Comment:
Shall we make the structure as use case 1/2/3 and etc., so that we can add more in the future? For example:
- use case 1: the one you wrote, each team has `1 Jira board`, `x Git repos`, and each deployment is performed within a repo
- use case 2: a more complex combination, 
   - each team maintains `X Jira boards for requirements`, but also shares `Y Jira boards for bugs and incidents`.
   - each team maintains `X repos for mainapps`, but also shares some `Y repos for libraries`.
   - each team maintains their own deployments for mainapps
 - use case 3: a case that incidents and deployments are both from Webhook
 - etc.

Reply:
Yup, that looks better structured, on-spot, and extensible. Thanks!